docs/tutorials/3 Advanced Bindings.md

```diff
@@ -258,9 +258,9 @@ To ensure complex filtering support is enabled the model is initialised with `si
 "": {
 	"type": "sap.fhir.model.r4.FHIRModel",
 	"dataSource": "fhir",
-    "settings":{
-      "simpleFiltering": false
-   }            
+        "settings":{
+          "simpleFiltering": false
```

Comment:
simpleFiltering should be never used anymore in the code and docu, isn't?

Reply:
no idea why this change came up here. In the changed files section the code looks correct.